In [8]:
import numpy as np

Parameters for argon gas under 300 Kelvin

In [9]:
NA = 6.02214e23  #/mol
T = 300  #K
rho = 1.784  #kg/m^3
N = 200
M = 0.039948  #kg/mol

In [10]:
#Calculate the cell length
m = (N / NA) * M  #kg
V = (m / rho) * (1e27)  #nm^3
L = V**(1.0 / 3.0)  #nm
print(L)  #nm
print(L / (200**0.33))  #Average distance on one axis of each atom

19.519099582236738
3.3971903454338084


Increase the pressure, cell length set to be 6 nm to test Langevin thermostat. 

In [11]:
kB = 1.380649e-23  #J/K
ep = kB * 120.0  #J
print(ep)

1.6567788e-21


The Lennard-Jones parameters are: $\sigma = 0.34\ nm$, $\frac{\epsilon}{k_B}= 120\ K$, $\epsilon = 1.66 \times 10^{-21}\ J$

In [12]:
print(ep*1e18*1e-30)

1.6567788000000003e-33


In [13]:
print(ep*NA/1000.0)

0.9977353882632


In [14]:
ep2 = 3.2135 #meV
ep2 *= (1e-3) * (1.60218e-19)#J
print(ep2*1e18*1e-30)


5.148605430000001e-34


In [15]:
ep2 = 3.2135  #meV
ep2 *= (1e-3) * (1.60218e-19)  #J
print(ep2*NA/1000.0)

0.31005622704220204
